advocacy_docs/supported-open-source/patroni/01-supported_topology.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: 'Supported Topology'
+description: 'EDB supported topology'
+tags:
+  - topology
+  - Patroni
+---
+
+### Description
+
+Main HA components:
+* PostgreSQL cluster: the database cluster, usually consisting of a primary and two or more replicas
+* Patroni: used as the failover management utility
+* etcd: used as a distributed configuration store (DCS), containing cluster information such as configuration, health, and current status
+
+![Supported topology](images/supported-arch.png)
+
+#### PostgreSQL
+
+EDB currently only supports using Patroni and PostgreSQL _Streaming Replication_ (physical) with a primary server and two or more replicas.
```

Comment:
"using Patroni for managing a PostgreSQL physical _Streaming Replication_ cluster with a primary and two replicas." (maybe even placing the "Physical" inside the italic block).
We are also only supporting 2 replicas as of now, and nothing fancy. Both replicas connected to the primary node.
@tomjohnkinedb @kelpoole  does this sound ok?

Reply:
changed to _EDB currently only supports using Patroni and Postgres _physical Streaming Replication_ with a primary server and two or more replicas (replicating directly from the primary)._ in 1fa7248